examples/grpc-client/client.go

```diff
@@ -49,7 +49,7 @@ func main() {
 	flag.Parse()
 
 	if *clientCrt == "" || *caCrt == "" || *clientKey == "" {
-		log.Fatal("Please provide CA & client certificates and client key. Usage: ./client -ca_crt=<path ca.crt> -client_crt=<path client.crt> -client_key=<path client key>")
+		log.Fatal("Please provide CA & client certificates and client key. Usage: ./client -ca-crt=<path ca.crt> -client-crt=<path client.crt> -client-key=<path client key>")
```

Comment:
while you're at it can you use two dashes for multi-letter command line flags? e.g. `--ca-crt`